src/classes/loggers/chalk-logger-decorator.ts

```diff
@@ -0,0 +1,28 @@
+import chalk from 'chalk';
+import {Option} from '../../enums/option';
+import {Logger} from './logger';
+
+/**
+ * @description Pretiffy log messages using chalk api
+ */
+export class ChalkLoggerDecorator extends Logger {
```

Comment:
Since chalk is only related to the "utils" we use right now I would prefer to have a generic name like
```suggestion
export class StylingLoggerDecorator extends Logger {
```
so that if chalk is removed it still have some logic in the naming